spec/presenter_spec.rb

```diff
@@ -1,24 +1,81 @@
 require "spec_helper"
 require "tictactoe"
+require "ostruct"
 
-RSpec.describe Presenter do
-  describe "#display_board" do
-    it "formats the board" do
-      presenter = Presenter.new
-      board = Board.new
-      board.place_token(5)
-      blank_string = " "
-      expected_board = <<~BOARD
+RSpec.describe TextPresenter do
+  subject(:presenter) { TextPresenter.new }
 
-           |   | #{blank_string}
-        ---+---+---
-           | X | #{blank_string}
-        ---+---+---
-           |   | #{blank_string}
+  class TextPresenter::FakeBoard
```

Comment:
I liked our other name a little better (`BoardWithOutcome`), but this is ok too. No change needed 😄